gap/CompilerLogic.gi

```diff
@@ -4,6 +4,8 @@
 # Implementations
 #
 
+CapJitAddTypeSignature( "/", [ IsInt, IsInt ], IsInt );
```

Comment:
This is not true in general. Please either use `QuoInt` or introduce a new function `SafeIntegerDivision` (or similar) to CAP, which asserts that the result of the division is an integer.

Reply:
Deleted.

Reply:
> It thought it is safe for this package, but maybe you are thinking of loading this file when compiling packages depending on `SkeletalFinSets`.

Exactly. If you need a hack like this in the future, please put it in the `Precompile*.g` files.

Reply:
Yes, good idea.